tests/integration/testdata/sync/infra/template-python-after.yaml

```diff
@@ -11,7 +11,7 @@ Resources:
     Properties:
       CodeUri: after/Python/function/
       Handler: app.lambda_handler
-      Runtime: python3.7
+      Runtime: python3.8
```

Comment:
The 3.8 changes were reverted in this commit https://github.com/aws/aws-sam-cli/commit/6231a837552ef99fe02d486a38c00be831aad9f1 should check with Mohamed on the reason for sticking with python 3.7

Reply:
Reverted back to 3.7. Adding a backlog item to update Windows tests to 3.8.